### PR TITLE
Fix undefined variable

### PR DESCRIPTION
**Pull request by nickvergessen**
:see_no_evil:

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

**Comment by nickvergessen** [comment]
More to come....

File: build/integration/features/bootstrap/CommandLine.php [hunk 1]
```diff
@@ -106,7 +106,7 @@ public function findLines($input, $text) {
 		$results = [];
 		// the exception text usually appears after an "[Exception"] row
 		foreach (explode("\n", $input) as $line) {
-			if (strpos($line, $text) >= 0) {
+			if (strpos($line, $text) !== false) {
```
Comment (nickvergessen) [inline build/integration/features/bootstrap/CommandLine.php:109]: This actually said it always matched

Comment (PVince81) [reply build/integration/features/bootstrap/CommandLine.php:109]: yeah, PHP not JS...

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by nickvergessen** [comment]
Woop woop
Will fix tomorrow

File: build/integration/features/transfer-ownership.feature [hunk 2]
```diff
@@ -547,19 +547,19 @@ Feature: transfer-ownership
...
-		Then the command error output contains the text "Unknown target user"
+		Then the command output contains the text "Unknown destination user invalid_user"
 		And the command failed with exit code 1
 
 	Scenario: transferring ownership fails with invalid path
 		Given user "user0" exists
 		And user "user1" exists
 		When transferring ownership of path "test" from "user0" to "user1"
-		Then the command error output contains the text "Unknown target user"
+		Then the command output contains the text "Unknown path provided: test"
```
Comment (nickvergessen) [inline build/integration/features/transfer-ownership.feature:564]: Invalid path should return an unknown target user? I dont think so...

File: build/integration/features/transfer-ownership.feature [hunk 3]
```diff
@@ -547,19 +547,19 @@ Feature: transfer-ownership
...
-		Then the command error output contains the text "Unknown source user"
+		Then the command output contains the text "Unknown source user"
 		And the command failed with exit code 1
 
 	Scenario: transferring ownership fails with invalid target user
 		Given user "user0" exists
 		And User "user0" created a folder "/sub"
 		When transferring ownership of path "sub" from "user0" to "invalid_user"
-		Then the command error output contains the text "Unknown target user"
+		Then the command output contains the text "Unknown destination user invalid_user"
```
Comment (nickvergessen) [inline build/integration/features/transfer-ownership.feature:557]: https://github.com/nextcloud/server/blame/0fab27f459d3276a2559cdee5693bfec8c2cf1c9/apps/files/lib/Command/TransferOwnership.php#L123

Always has been the destination user not the target user

File: build/integration/features/transfer-ownership.feature [hunk 4]
```diff
@@ -297,13 +297,13 @@ Feature: transfer-ownership
 	Scenario: transferring ownership fails with invalid source user
 		Given user "user0" exists
 		When transferring ownership from "invalid_user" to "user0"
-		Then the command error output contains the text "Unknown source user"
+		Then the command output contains the text "Unknown source user"
```
Comment (nickvergessen) [inline build/integration/features/transfer-ownership.feature:300]: all our output is on stdout not stderr

File: build/integration/features/bootstrap/CommandLine.php [hunk 5]
```diff
@@ -158,19 +141,13 @@ public function theCommandFailedWithException($exceptionText) {
 	 * @Then /^the command output contains the text "([^"]*)"$/
 	 */
 	public function theCommandOutputContainsTheText($text) {
-		$lines = $this->findLines($this->lastStdOut, $text);
-		if (empty($lines)) {
-			throw new \Exception('The command did not output the expected text on stdout "' . $exceptionText . '"');
-		}
```
Comment (nickvergessen) [inline build/integration/features/bootstrap/CommandLine.php:164]: Not sure why something complicated was tried, when there is a function to check for a string already.

File: build/integration/features/transfer-ownership.feature (hunk 2 above)
Comment (PVince81) [reply build/integration/features/transfer-ownership.feature:564]: hmm how could the test pass before ? did it run at all ?

Comment (PVince81) [reply build/integration/features/transfer-ownership.feature:564]: ah, got it... it "always matched"

**Review by PVince81 — APPROVED** [review]
:+1:

**Review by rullzer — APPROVED** [review]
:see_no_evil:

**Pull request merged**